flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -1772,9 +1773,23 @@ private StateBackend loadStateBackend(ReadableConfig configuration, ClassLoader
                         + ContinuousFileMonitoringFunction.MIN_MONITORING_INTERVAL
                         + " ms.");
 
+        int readerParallelism = getParallelism();
+
+        boolean isAdaptiveBatchSchedulerEnabled =
```

Comment:
Maybe `useDynamicGraphSourceParallelism` is more accurate because conceptually `isAdaptiveBatchSchedulerEnabled == getConfig().isDynamicGraph()`.

Reply:
yes, `useDynamicGraphSourceParallelism` is more appropriate here, I will fix it